docs/agents/sematext-agent/kubernetes/installation.md

```diff
@@ -234,3 +291,31 @@ Deploy the DaemonSet:
 ```sh
 kubectl apply -f st-agent-ds.yml
 ```
+
+### Updating/Upgrading Sematext Agent using Manual Installation
+
+To update the Sematext Agent when installed manually, you need to modify the DaemonSet configuration and apply it:
+
+1. **Edit the DaemonSet YAML file:** Update the image to the latest version or change any configuration.
+
+```yaml
+    - name: agent
+      image: sematext/agent:latest
+```
+
+2. **Apply the updated DaemonSet:**
+
+```sh
+    kubectl apply -f st-agent-ds.yml
+```
+
+### Uninstalling Sematext Agent using Manual Installation
+
+To uninstall the Sematext Agent when installed manually, delete the DaemonSet and related resources:
+
+```sh
+kubectl delete -f st-agent-ds.yml
```

Comment:
This won't remove Vector pods, fyi. IMO better to apply commands from SC.

Orphan Vector issue will be fixed in 3.7.0 but we have the issue today.